meetings/serializers.py

```diff
@@ -37,23 +40,39 @@ class Meta:
                   "title",
                   "organizer",
                   "room",
+                  "users",
                   "external",
-                  "created_by",
                   "date_from",
                   "date_to",
                   ]
-        read_only_fields = ('created_by',)
 
-    def validate_title(self, data):
-        if 'title' == '':
+    def validate_users(self, data):
+        print(len(data))
+        if len(data) >= 4:
             raise ValidationError(
-                {'password': 'password_no_upper'},
+                {'error': 'to_many_users_in_room'},
+            )
+        return data
+
+    def validate(self, data):
+        room = data['room']
+        check_in = data['date_from']
+        check_out = data['date_to']
+
+        check_room_time = Reservation.objects.filter(room=room).filter(
+            Q(date_from__lte=check_in, date_to__gte=check_in) |
```

Comment:
as we spoke - let's add `room=room` case into this filter, so merge it with existing Q expression